pkg/types/vsphere/validation/platform.go

```diff
@@ -94,3 +100,21 @@ func validateFolder(p *vsphere.Platform, fldPath *field.Path) field.ErrorList {
 
 	return allErrs
 }
+
+// ValidateClient validates vCenter to only contain hostname and port
+func ValidateClient(vCenter string) error {
```

Comment:
i think this needs a better name, like ValidateVCenterAddress or something like that...